packages/control-property-editor/package.json

```diff
@@ -0,0 +1,105 @@
+{
+  "name": "@sap-ux/control-property-editor",
+  "displayName": "SAP Fiori Tools - Control Property Editor",
```

Comment:
You are not in the Fiori tools repo anymore ;)

Reply:
:) Removed it